lib/base-container.js

```diff
@@ -94,4 +94,8 @@ export default class BaseContainer {
 
 		return this.waitForPage();
 	}
+
+	navigateBack() {
```

Comment:
I think this would make more sense to go in `driver-helper` rather than `base-container`